app/src/ereferrals/java/org/eyeseetea/malariacare/fragments/AVFragment.java

```diff
@@ -66,7 +67,7 @@
     CustomTextView mTextProgressView;
     private CustomTextView mErrorMessage;
     IMainExecutor mainExecutor;
-    IMediaModeRepository stylePreferencesRepository;
+    ISettingsRepository settingsDataSource;
```

Comment:
a Fragment should not use the settingsDataSource, It's the presenter who should invoke request to retrieve data to data layer but through of a use case, please create also GetSettingsUseCase if not exists